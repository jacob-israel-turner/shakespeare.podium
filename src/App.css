* {
  box-sizing: border-box;
}

.review-table td, th {
  border: 1px solid #888;
}
