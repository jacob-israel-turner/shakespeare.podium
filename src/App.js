import React, { Component } from 'react'
import {ReviewList} from 'components'
import {StyleSheet, css} from 'aphrodite'

import './App.css'

const styles = StyleSheet.create({
  container: {
  }
})

class App extends Component {
  render() {
    return (
      <div className={css(styles.container)}>
        <ReviewList />
      </div>
    );
  }
}

export default App;
