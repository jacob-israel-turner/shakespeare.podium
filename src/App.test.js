import React from 'react';
import App from './App'
import renderer from 'react-test-renderer'
import {StyleSheetTestUtils} from 'aphrodite'

beforeEach(() => {
  StyleSheetTestUtils.suppressStyleInjection();
});

afterEach(() => {
  StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
});

describe('Shakespeare', () => {
  it('renders without crashing', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  });
})
