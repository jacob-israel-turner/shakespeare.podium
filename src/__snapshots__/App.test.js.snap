exports[`Shakespeare renders without crashing 1`] = `
<div
  className="container_120drhm">
  <div
    className="container_py3uvl">
    <h3>
      Loading...
    </h3>
  </div>
</div>
`;
